cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -694,24 +690,25 @@
     <property>
         <name>router.bind.address</name>
         <value>0.0.0.0</value>
-        <description>Specifies the address for router server to bind to</description>
+        <description>Router server address</description>
     </property>
 
     <property>
         <name>router.webapp.enabled</name>
         <value>false</value>
-        <description>Specifies if webapp listening service should be started at Router</description>
+        <description>Specifies if webapp listening service should be started at router</description>
```

Comment:
Fixed. Made all the "enable" properties identical. However, the description of this property is baroque.